src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -540,3 +540,11 @@ def write_metadata_file(final_output_dir: Path, label_version: str) -> None:
     outpath = final_output_dir / "metadata.yaml"
     with open(outpath, "w") as file:
         yaml.dump(metadata, file)
+
+
+def load_nicknames_data():
+    # Load and format nicknames dataset
+    nicknames = pd.read_csv(paths.NICKNAMES_DATA_PATH)
+    nicknames = nicknames.apply(lambda x: x.astype(str).str.title()).set_index("name")
```

Comment:
This can be simplified to just `nicknames = nicknames.apply(lambda x: x.str.title()).set_index("name")`

Anything that didn't work will already be cast as Nonetype so no need for replacement.

Reply:
But we want them to be NaNs instead of Nonetype. I'm mainly just trying to be consistent with handling missing values as NaNs to avoid any confusion but I don't think it really matters.

Reply:
I meant NaN.